time msg_time
int32 image_width
int32 image_height

int32 lane_left_max_x
int32 lane_left_max_y
int32 lane_left_min_x
int32 lane_left_min_y

int32 lane_right_max_x
int32 lane_right_max_y
int32 lane_right_min_x
int32 lane_right_min_y

